### ISSUE TITLE
Adding &optional to zeek records question

### ISSUE THREAD

**Issue by zrobinette12** [FoxIO-LLC/ja4#135]
Hello, was there a specific reason to not use `&optional` on several of the ja4 zeek records such as here? https://github.com/FoxIO-LLC/ja4/blob/main/zeek/ja4t/main.zeek#L42.

When writing JSON logs, it ends up looking like this in the JSON message if there's no value for JA4T and others that may not get recorded for a specific connection. `"ja4t": ""`.

When adding `&optional`, the key/value just doesn't get written out instead of it being there but empty.

**Comment by john-althouse** [comment]
@J0eJ0h I tend to agree with this, no reason to log a blank field. What do you think?
